typings/index.d.ts

```diff
@@ -2962,6 +2966,12 @@ declare module 'discord.js' {
 
   type RoleResolvable = Role | string;
 
+  interface RoleTagData {
+    botID?: Snowflake;
+    integrationID?: Snowflake;
+    premiumSubcriberRole?: boolean;
```

Comment:
```suggestion
    premiumSubscriberRole?: true;
```
Would make more sense because it's either `true` or not there at all (also fixes typo)